src/core/meshcop/border_agent.cpp

```diff
@@ -560,12 +560,14 @@ void BorderAgent::EphemeralKeyManager::SetEnabled(bool aEnabled)
     {
         VerifyOrExit(mState == kStateDisabled);
         SetState(kStateStopped);
+        Get<MeshCoP::BorderAgent>().PostNotifyMeshCoPServiceChangedTask();
```

Comment:
I think we don't need the `MeshCoP::` (we should be already under this namespace).
```suggestion
        Get<BorderAgent>().PostNotifyMeshCoPServiceChangedTask();
```